lib/mu-msg-sexp.c

```diff
@@ -445,6 +446,25 @@ append_message_file_parts (GString *gstr, MuMsg *msg, MuMsgOptions opts)
 	if (str || (str = mu_msg_get_header (msg, "X-Mailer")))
 		append_sexp_attr (gstr, "user-agent", str);
 
+        params = mu_msg_get_body_text_content_type_parameters(msg, opts);
+        if (params) {
+		GString *paramstr = g_string_new("");
```

Comment:
Please split declaration + initialization into separate lines. Also, use tabs for indentation, spaces for alignment.